fiftyone/core/annotation/validate_label_schemas.py

```
@@ -696,6 +726,14 @@ def _validate_values_setting(field_name, value, _type, key=foac.VALUES):
             )
 
 
+def _validate_taxonomy_setting(field_name, taxonomy):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_validate_taxonomy_setting`

Add return type annotation: `None`

(ANN202)